esp/esp/themes/theme_data/circles/templates/main.html

```diff
@@ -113,7 +113,7 @@
 </div>
 
 {% load preview %}
-{% miniblog_for_user None as announcements 6 %}
+{% miniblog_for_user AnonymousUser as announcements 6 %}
```

Comment:
I'm not sure what this is supposed to be doing, but I'm wondering if it does what was intended or something else due to #1519